postgrest.cabal

```diff
@@ -44,8 +44,8 @@ executable postgrest
                      , http-types
                      , interpolatedstring-perl6
                      , jwt
-                     , microlens >= 0.4.2 && < 0.5
-                     , microlens-aeson >= 2.1.1 && < 2.2
+                     , lens >=3.8 && < 5.0
```

Comment:
lens was dropped in favore of microlense becasue it pull is a lot of dependencies. Can you refactor your code to only use microlense?